chapters/annotations.tex

```diff
@@ -114,12 +114,16 @@ \subsection{Annotations for Figures\doublelabel{annotations-for-figures}}
 replacements, as described in \autoref{variable-replacements}, can be used in the
 \lstinline!title! of \lstinline!Figure! and \lstinline!Plot!.
 
-The \lstinline!identifier! in \lstinline!Figure! and \lstinline!Plot! is
-optional, and is intended for programmatic access.  A \lstinline!Figure!
-\lstinline!identifier! must be unique within the class containing the
-\lstinline!figures! annotation, whereas a \lstinline!Plot!
-\lstinline!identifier! must be unique among the \lstinline!plots! in the
-\lstinline!Figure! annotation.
+The \lstinline!identifier! in \lstinline!Figure! and \lstinline!Plot! is an
+optional \lstinline!String! identifier, and is intended to identify the
```

Comment:
Is it necessary to say it's a `String` here, given the pseudo-code record definitions?

Reply:
The confusion was about the identifiers living in the same namespace as everything else, spelling out String here should make that more clear.

Reply:
OK, let's see if it the kind of clarification @HansOlsson was looking for.

Reply:
Ok, I think it is fine now.